src/diffusers/pipelines/ddpm/pipeline_ddpm.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, unet, scheduler):
         scheduler = scheduler.set_format("pt")
         self.register_modules(unet=unet, scheduler=scheduler)
 
-    def __call__(self, batch_size=1, generator=None, torch_device=None):
+    def __call__(self, batch_size=1, generator=None, torch_device=None, num_inference_steps=1000):
```

Comment:
```suggestion
    def __call__(self, batch_size=1, generator=None, torch_device=None):
```

Reply:
DDPM is known to have always done a 1000 steps so let's just force here. It makes a nice story DDPM -> 1000 steps, good quality -> next paper: DDIM -> 50 steps, much faster almost same quality

cc @anton-l